packages/quantic/force-app/main/default/lwc/quanticTimeframeFacet/quanticTimeframeFacet.html

```diff
@@ -5,121 +5,145 @@
   </template>
   <template lwc:else>
     <slot><!-- Timeframes --></slot>
+    <template lwc:if={enabled}>
+      <template if:true={showPlaceholder}>
+        <c-quantic-placeholder
+          variant="card"
+          number-of-rows="5"
+        ></c-quantic-placeholder>
+      </template>
+      <template if:false={showPlaceholder}>
+        <template if:true={showFacet}>
+          <div class="slds-size_1-of-1 slds-var-m-bottom_large">
+            <c-quantic-card-container
+              title={label}
+              onquantic__headerclick={toggleFacetVisibility}
+              onquantic__headerkeydown={toggleFacetVisibility}
+            >
+              <lightning-button-icon
+                slot="actions"
+                class={actionButtonCssClasses}
+                icon-name={actionButtonIcon}
+                alternative-text={actionButtonLabel}
+                onmousedown={preventDefault}
+                variant="bare"
+                tabindex="-1"
+                aria-hidden="true"
+              ></lightning-button-icon>
 
-    <template if:true={showPlaceholder}>
-      <c-quantic-placeholder variant="card" number-of-rows="5"></c-quantic-placeholder>
-    </template>
-    <template if:false={showPlaceholder}>
-      <template if:true={showFacet}>
-        <div class="slds-size_1-of-1 slds-var-m-bottom_large">
-          <c-quantic-card-container title={label} onquantic__headerclick={toggleFacetVisibility} onquantic__headerkeydown={toggleFacetVisibility}>
-            <lightning-button-icon
-              slot="actions"
-              class={actionButtonCssClasses}
-              icon-name={actionButtonIcon}
-              alternative-text={actionButtonLabel}
-              onmousedown={preventDefault}
-              variant="bare"
-              tabindex="-1"
-              aria-hidden="true"
-            ></lightning-button-icon>
+              <template if:true={hasActiveValues}>
+                <button
+                  class="facet__clear-filter slds-button slds-grid slds-grid_vertical-align-center slds-p-horizontal_x-small slds-m-top_small"
+                  onclick={clearSelections}
+                  title={clearFilterLabel}
+                  aria-label={clearFilterAriaLabelValue}
+                >
+                  <lightning-icon
+                    class="slds-current-color slds-m-right_xx-small"
+                    icon-name="utility:close"
+                    aria-hidden="true"
+                    size="xx-small"
+                  ></lightning-icon>
+                  <span class="pill__text-container slds-truncate"
+                    >{clearFilterLabel}</span
+                  >
```

Comment:
Yuck what did you run :D Is it prettier doing this? Putting the end `>` at a new line? lol

Reply:
Prettier 🤨 I will double check x)

Reply:
prettier officially does that x)

Reply:
yuck yea that's what I thought lol